contracts/dev/DevAdmin.sol

```diff
@@ -49,6 +53,11 @@ contract DevAdmin is AccessControlInternal {
         string attribute,
         string info
     );
+    event AftermarketDevicePaired(
```

Comment:
I see, just repeating.

Again, we'd like to upgrade this to a real thing at some point but this seems fine for now.